AppCenterAnalytics/AppCenterAnalyticsTests/MSLongTypedPropertyTests.m

```diff
@@ -0,0 +1,72 @@
+#import "MSLongTypedProperty.h"
+#import "MSConstants+Internal.h"
+#import "MSTestFrameworks.h"
+
+@interface MSBooleanTypedPropertyTests : XCTestCase
+
+@end
+
+@implementation MSBooleanTypedPropertyTests
+
+- (void)testNSCodingSerializationAndDeserializationWorks {
+
+  // If
+  MSLongTypedProperty *sut = [MSLongTypedProperty new];
+  sut.type = @"type";
+  sut.name = @"name";
+  sut.value = 12;
```

Comment:
Should use a value that wouldn't fit in a regular integer.  See if there's a `#define` for MAX_INT64` or similar, or use a value 2^32<x<2^64

Reply:
Also should move this into a constant since it's the same for each test.